rust/src/connection/network/channel.rs

```diff
@@ -37,73 +37,42 @@ use crate::{
 
 type ResponseFuture = InterceptorResponseFuture<ChannelResponseFuture>;
 
-pub(super) type PlainTextChannel = InterceptedService<Channel, PlainTextFacade>;
 pub(super) type CallCredChannel = InterceptedService<Channel, CredentialInjector>;
 
 pub(super) trait GRPCChannel:
     GrpcService<BoxBody, Error = TonicError, ResponseBody = BoxBody, Future = ResponseFuture> + Clone + Send + 'static
 {
 }
 
-impl GRPCChannel for PlainTextChannel {}
-
 impl GRPCChannel for CallCredChannel {}
 
-pub(super) fn open_plaintext_channel(address: Address) -> PlainTextChannel {
```

Comment:
Why do we remove this? I don't know what it was used for.

Reply:
Because it's no longer used. In 3.0 both Core and Cloud would make use of the callcred channel, which is the channel type you need to use if you want authentication.